### ISSUE TITLE
Example code greet handler uses current_user variable which is not defined at that point

### ISSUE THREAD

**Issue by taladar** [AscendingCreations/AxumSqlxSessionAuth#17]
Maybe that was supposed to be auth.current_user ? There is also a section inside that if where auth.current_user is retrieved again though.

The greet handler in general is hard to follow.

**Comment by genusistimelord** [comment]
Thank you for catching that .  I will update it soon. Also the reason its complex is its showing a few different ways to use Auth which includes Rights Management. probably can make it more clear with some comments.

**Comment by taladar** [comment]
Maybe it would make sense to have one handler per feature you want to demonstrate to delineate better where one example usage ends and the next one starts?

Thanks for your quick response.

**Comment by genusistimelord** [comment]
let me know if the new readme greet is a better example for you as well.

**Comment by taladar** [comment]
The comments certainly help me understand what is going on. Thanks.
